adventure/api.py

```diff
@@ -63,5 +63,13 @@ def move(request):
 @csrf_exempt
 @api_view(["POST"])
 def say(request):
-    # IMPLEMENT
-    return JsonResponse({'error':"Not yet implemented"}, safe=True, status=500)
+    player = request.user.player
+    player_id = player.id
+    room = player.room()
+    data = json.loads(request.body)
+    message = data['message']
+    currUsers = room.playerNames(player_id)
+    currPlayerIDs = room.playerUUIDs(player_id)
+    for p_id in currPlayerIDs:
+        pusher.trigger(f'p-channel-{p_id}', u'broadcast', {'message': f'{player.user.username} grumbles to you {message}'})
+    return JsonResponse({'name':player.user.username, 'title':room.title, 'description':room.description, 'players':currUsers}, safe=True)
```

Comment:
Good structure here! The way you pass data around here will make your job easier hooking up front and back.